README.md

```diff
@@ -479,12 +482,18 @@ OPTIONS
   -t, --templates=templates
       Path to the templates folder
 
+  -v, --version=version
```

Comment:
I think it's odd because I might expect that -v or --version will display the current version of the cli

Reply:
The flag is used as an argument for `server:deploy` command, so it will not collide with `-v` flag of `chectl` itself.